lib/rules/no-useless-computed-key.js

```diff
@@ -33,7 +35,25 @@ module.exports = {
                     nodeType = typeof key.value;
 
                 if (key.type === "Literal" && (nodeType === "string" || nodeType === "number")) {
-                    context.report(node, MESSAGE_UNNECESSARY_COMPUTED, { property: sourceCode.getText(key) });
+                    context.report({
+                        node,
+                        message: MESSAGE_UNNECESSARY_COMPUTED,
+                        data: { property: sourceCode.getText(key) },
+                        fix(fixer) {
+                            const leftSquareBracket = sourceCode.getTokenBefore(node.key);
+                            const rightSquareBracket = sourceCode.getTokenAfter(node.key);
```

Comment:
Whoops, yes, that does cause issues; it causes the parens to be removed and not the square brackets. Thanks, I'll figure out how to fix that.

(Technically it doesn't cause incorrect behavior overall since the square brackets would be removed on the next pass, but it's worth fixing regardless.)